### PR TITLE
Disable a regex test that is failing with minopts

### PR DESCRIPTION
**Pull request by kouvel**
See https://github.com/dotnet/coreclr/issues/18912. It looks like corefx test runs are being triggered from PRs in coreclr, disabling because this would always fail once tiered compilation is enabled by default.

### CODE REVIEW

**Comment by ViktorHofer** [comment]
Can you Summarize why this is failing now with JitMinOps?

**Comment by kouvel** [comment]
It's timing out, there's some info in the issue but I don't know why it's timing out.

**Comment by kouvel** [comment]
Some investigation was done, there's more info in the issue, however it's not clear yet why it's failing in the CI and someone would need to investigate. It's only failing in the CoreCLR repo, so closing this in favor of https://github.com/dotnet/coreclr/pull/19488.
